### DISCUSSION TITLE
Difficulty  logging in, depending on browser; unable to set up 2FA

### DISCUSSION THREAD

**Discussion by kogorman** [community/community#75227] [New to GitHub] [answered]
### Login is working in Chrome, but not in Firefox

On the same machine, running Xubuntu Linux, I can log in (as I am now) with Chrome.  I am unable on Firefox because it complains that the username or password is incorrect.  What's odd about this is that in both cases, I'm using Roboform to do the logging in, and when I use the "view" option in Roboform on both browsers, the credentials are identical.

It also happens that I cannot sign on with my iPhone with Roboform either, even with the GitHub app.

So this is my first question: what's going on here?

The second question is "Why is it so complicated to set up 2FA?" .  I've tried setting up Google Authenticator to no avail.  Then I tried using text messages but setting that up asks me for password, and I can't get past that in either browser.  I'm hoping that a clear answer to the first question will answer the second question or make it moot.

The good news is that I still have 15 of my recovery codes.  So I can work on this.

**Comment by Neeham** [comment, marked as answer]
Hey @kogorman,

It happens that sometime extensions like Roboform, LastPass, Bitwarden enter incorrect or out of sync data.
Ensure that on your Firefox, the extension is synchronized  and upto date. It could be you changed the password and it was updated on the extension found on Chrome but not on the one found in Firefox.

Another theory could be that even though you click on Roboform to fill in your credential, it's possible Firefox already has the old credentials saved and it uses that version. This is why I recommend you try with firefox in incognito/private mode and see if this issue reproduces.
